test/jdk/TEST.groups

```diff
@@ -598,26 +598,29 @@ jdk_nio_networkchannel = \
 
 jdk_core_manual = \
     :jdk_core_manual_no_input \
```

Comment:
jdk_core_manual_human (as it is now called) should be part of jdk_core_manual, as jdk_core_manual will be used in the corelibs-atr job